docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -229,24 +261,43 @@ spec:
-|-|-|-|
-| filters | []object | A list of [Fluent Bit filters](https://docs.fluentbit.io/manual/pipeline/filters) to apply to the logs processed by the pipeline. Filters are executed in sequence, as defined. They are executed before logs are buffered, and with that, are not executed on retries.|
-| filters[].custom | string | The actual filter definition in the syntax of Fluent Bit.|
-| output | object | [Fluent Bit outputs](https://docs.fluentbit.io/manual/pipeline/outputs).|
-| output.custom | string | The actual output definition in the syntax of Fluent Bit.|
+|---|---|---|
+| filters | []object | List of [Fluent Bit filters](https://docs.fluentbit.io/manual/pipeline/filters) to apply to the logs processed by the pipeline. Filters are executed in sequence, as defined. They are executed before logs are buffered, and with that, are not executed on retries.|
+| filters[].custom | string | Actual filter definition in the Fluent Bit syntax.|
+| output | object | [Fluent Bit output](https://docs.fluentbit.io/manual/pipeline/outputs) where the logs should be pushed to. Only one output can be specified. |
+| output.http | object | [Fluent Bit http output](https://docs.fluentbit.io/manual/pipeline/outputs/http). |
+| output.http.compress | string | Payload compression mechanism. |
+| output.http.dedot | boolean | Indicates if *dedotting* (replacing dots with underscores) should be applied on the log field names `kubernetes.annotations` and `kubernetes.labels`. Default is `false`. |
+| output.http.format | string | Data format to be used in the HTTP request body. Default is `json`. |
+| output.http.host | object | IP address or hostname of the target HTTP server. |
+| output.http.host.value | string | Actual host value, can contain references to secret values. |
+| output.http.host.valueFrom.secretKeyRef | object | Reference to a key in a Secret. `name` and `namespace` of the Secret, as well as the name of the `key`, must be provided. |
+| output.http.password | object | Basic Auth password. |
+| output.http.password.value | string | Actual password value, can contain references to secret values. |
+| output.http.password.valueFrom.secretKeyRef | object | Reference to a key in a Secret. `name` and `namespace` of the Secret, as well as the name of the `key`, must be provided. |
+| output.http.port | string | TCP port of the target HTTP server. Default is `443`.  |
+| output.http.tls | object | TLS Configuration of the HTTP target server.  |
+| output.http.tls.disabled | boolean | Indicates if TLS is disabled or enabled. Default is `false`. |
+| output.http.tls.skipCertificateValidation | boolean | Indicates if the validation of certificates should be skipped. Default is `false`. |
+| output.http.uri | string | URI for the target HTTP server. Fluent Bit Default is `/`. |
+| output.http.user | object | Basic Auth username.|
+| output.http.user.value | string | Actual username value, can contain references to secret values. |
```

Comment:
```suggestion
| output.http.user.value | string | Username value. Can contain references to secret values. |
```

Is this a resource "Secret", which should be [capitalized](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology#capitalization) (like line 285); or just a normal kind of secret in lower case?